imagemounter/volume_system.py

```diff
@@ -279,6 +305,11 @@ def _detect_mmls_volumes(self, vstype='detect'):
                 except Exception as e:
                     logger.exception("Failed executing mmls command")
                     raise SubsystemError(e)
+            # since mmls cannot determine the partition type for exFAT superfloppy formatting; check for this.
+            elif self._detect_exfat_superfloppy():
```

Comment:
I agree and have removed the EXFAT detection method and condition.  I have been using the imagemounter in auto detection mode, which in my case wouldn't bother with parted.  I'll commit these along with the changes from the other comments and push to my fork (I think this will update the pull request).